corehq/apps/registration/utils.py

```diff
@@ -86,9 +86,13 @@ def request_new_domain(request, form, domain_type=None, new_user=True):
 
     if not new_domain.name:
         new_domain.name = new_domain._id
-        new_domain.save() # we need to get the name from the _id
+        new_domain.save()  # we need to get the name from the _id
+
+    if new_user:
+        create_30_day_trial(new_domain)
+    else:
```

Comment:
But it looks like `create_30_day_trial` is creating an Advanced plan. For old - users we want to create a Community plan, so we can't remove the else block, right?